tests/Functional/Controller/BackPartnerControllerTest.php

```diff
@@ -53,10 +147,35 @@ public function testDeleteUserAccount(): void
         $user = $userRepository->findOneBy(['email' => 'user-974-01@histologe.fr']);
         $userId = $user->getId();
 
-        $crawler = $client->request('POST', $router->generate('back_partner_user_delete', [
-            'user' => $userId,
+        $client->request('POST', $router->generate('back_partner_user_delete', [
+            'user_delete[user]' => $userId,
+            '_token' => $this->generateCsrfToken($client, 'partner_user_delete'),
+        ]));
```

Comment:
$client->request('POST', $router->generate('back_partner_user_delete'), [
            'user_id' => $userId,
            '_token' => $this->generateCsrfToken($client, 'partner_user_delete'),
        ]));